.travis.yml

```diff
@@ -28,7 +28,6 @@ before_script:
 script:
 - cargo build
 - cargo test
-- cargo fmt -- --check
```

Comment:
Why did we remove this check?

Reply:
my bad, it was meant for the other [pallet](https://travis-ci.com/github/centrifuge/substrate-pallet-nft-registry) which ci was causing errors(which I thought was `cargo fmt -- --check`).